src/Model/Item.php

```diff
@@ -342,7 +343,7 @@ public static function markForDeletionById($item_id, $priority = PRIORITY_HIGH)
 
 		Post\Category::storeTextByURIId($item['uri-id'], $item['uid'], '');
 
-		if (!Post::exists(["`uri-id` = ? AND `uid` != 0 AND NOT `deleted`", $item['uri-id']])) {
+		if (!Post::exists(['`uri-id` = ? AND `uid` != 0 AND NOT `deleted`', $item['uri-id']])) {
```

Comment:
The convention is to use double-quote for SQL statements or conditions.
```suggestion
		if (!Post::exists(["`uri-id` = ? AND `uid` != 0 AND NOT `deleted`", $item['uri-id']])) {
```